pkg/sdn/plugin/registry.go

```diff
@@ -263,6 +263,34 @@ func (registry *Registry) GetEgressNetworkPolicies() ([]osapi.EgressNetworkPolic
 	return policyList.Items, nil
 }
 
+// Returns true to indicate status update is needed
+type UpdateNodeFunc func(node *kapi.Node) bool
```

Comment:
@liggitt we don't have access to the kubernetes client from the OsdnNode object, since we've been keeping the osClient/kClient in the Registry object.  I'd need more input from the team before making a decision to break that encapsulation.